django/core/management/commands/makemigrations.py

```diff
@@ -48,6 +48,14 @@ def add_arguments(self, parser):
             '-n', '--name',
             help="Use this name for migration file(s).",
         )
+        parser.add_argument(
+            '--no-version', action='store_true',
+            help="Do not write the Django version to the header comment in new migration file(s).",
+        )
+        parser.add_argument(
+            '--no-timestamp', action='store_true',
+            help="Do not write the current date and time to the header comment in new migration file(s).",
+        )
```

Comment:
To be honest it's because personally I'm OK with the version while I strip the date in a script. The version wouldn't be exactly knowable based on anything else we have in version control.

The date _always_ changes while the version changes more rarely.

Reply:
> As you said the usage of a fixed `PYTHONHASHSEED` might be easier than trying to tackle all the edge cases here for a the rare cases when this is required.
> 
> Sorry for pushing you towards working on a patch but I feel like it's not worth doing at this point.

I feel it could be considered worthwhile as a step in the direction of making migration files more determinate. Would it help to reword anything to make more clear that this only addresses the ordering of dependencies?